cmake/ExternalProjects.cmake

```diff
@@ -52,13 +63,13 @@ set_target_properties(pistache
 )
 
 # RapidJSON
-set(RAPIDJSON_BUILD_DOC OFF CACHE INTERNAL "")
```

Comment:
After double-checking, and inspecting the contents of `/build/faabric` after a clean sweep, this was actually not enforced. This other option through `CMAKE_CACHE_ARGS` seems to do it.